src/cascadia/TerminalControl/UiaTextRange.cpp

```diff
@@ -4,10 +4,12 @@
 #include "pch.h"
 #include "UiaTextRange.hpp"
 #include "TermControlUiaProvider.hpp"
+#include <winrt/Windows.Graphics.Display.h>

```

Comment:
should this go in `pch.h`?

Reply:
huh.... apparently, it already is... removing.